backend/accessibility_templates/aa-003.xml

```diff
@@ -0,0 +1,16 @@
+<AccessibilityAttributes>
+ <SubtitleAttributes>
+  <!-- WVTT -> contentType="text" codecs="wvtt" lang="en" -->
+  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:99"/>  <!--other-->
+  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:4"/>  <!--WebVTT-->
+  <SubtitleLanguage>fra</SubtitleLanguage>
+  <SuitableForTTS>false</SuitableForTTS>
+ </SubtitleAttributes>
```

Comment:
The subtitle purpose should be specified, in this case the best value seems to be translastion termId 1.

Reply:
Will add `<Purpose href="urn:tva:metadata:cs:SubtitlePurposeCS:2023:1"/>` to `aa-003.xml`